apps/workspaces/tasks.py

```
@@ -89,47 +89,75 @@ def run_import_export(workspace_id: int, export_mode = None):
+    """
+    Remove existing schedule if it exists
+    """
+    if advance_settings.schedule:
+        schedule = advance_settings.schedule
+        advance_settings.schedule = None
+        advance_settings.save()
+        schedule.delete()
 
 
-def export_to_sage300(workspace_id: int, triggered_by: ExpenseImportSourceEnum):
+def export_to_sage300(workspace_id: int, triggered_by: ExpenseImportSourceEnum, accounting_export_filters: dict = {}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)